u4-databases/features/uni-controller.js

```diff
@@ -0,0 +1,50 @@
+const { db } = require('../_services/firebase-initialized')
+
+const iterationList = async (req, res) => {
+  const query = db.collection('iterations').get()
+  const iteration = (await query).docs.map(doc => doc.data())
+  res.render('uni-iteration-list', { iteration })
+}
+
+const iterationDetails = async (req, res) => {
+  const code = req.params.code
+  const query = db.collection('iterations').doc(code).get()
+  const iteration = (await query).data()
```

Comment:
นี่แสดงถึงการวนซ้ำหนึ่งครั้ง คุณถูกต้องที่จะใช้เอกพจน์: `iteration`
This represents one iteration. You are correct to use the singular: `iteration`.